main.go

```diff
@@ -118,6 +117,45 @@ func createRepo(args []string) {
 	getRepo([]string{repoPath})
 }
 
+func deleteRepo(args []string) {
```

Comment:
Maybe add a check for delete permissions before going through the warning and confirmation.